voting.md

```diff
@@ -1,53 +1,17 @@
-
-It is suggested that the [Helios voting service](https://vote.heliosvoting.org), or a similar service, be used for conducting the election. This choice is to be made by the election officers. The election officers are then also responsible for setting up the service for use before the voting phase begins.
-
-It should be possible to vote at least three members (as the Steering Council has three seats). It is suggested that voting more than three members is possible, so that people who cannot or do not want to decide between two candidates they perceive as equal have the option to vote both.
-
-### Phase Three: Results
-
-After voting has finished, the election officers announce the results internally as soon as possible for them. A transition of power is to be prepared, including reassignment of roles and permissions. In parallel, a public statement in the form of a Discord announcement and a blog post is to be prepared and released.
-
-In the event of a tie that would cause one or more members to win a seat on the council while one or more members with the same amount of votes would not win a seat, i.e. if a tie occurs for places 3 and 4 (and beyond), or for places 2, 3 and 4 (and beyond), or for places 1, 2, 3 and 4 (and beyond), the result is to be determined from among the candidates randomly by an election officer.
-
-## Communication to the Outside
-
-It should be announced to the public that an election process has been initiated. Other than that, in order to not compromise the integrity of the vote, public statements about the election should be kept to a minimum until the election is over.
-
-## Considerations
-
-Before the election is started, ensure that the Grace Period role is taken care of. See more [info about that here.](./inactivity-policy#grace-period-at-steering-council-voting)
+# Team votes
+To ensure that members of a team are in agreement regarding a topic under their purview, a vote may be called by any member of the team or by a member of the Steering Council.  
+The votes take the form of Discord polls, sent to one of the following channels:
+- `#project-talk` for votes concerning all teams of the Project;
+- `#maintenance-talk` for votes concerning the Maintainers team;
+- `#internal-private` for votes that need to be held privately or concerning internal team structures;
+- `#moderation-private` for votes concerning the Moderators team.
+Any accompanying contextual information must be sent in messages directly below the poll, alongside a ping of the role of the team in question.
+
+Voting takes place in the 24 hours following the creation of the poll, unless the Steering Council agrees to lift waiting requirements for situations that require prompt resolutions. If this is the case, the Council must clearly explain the context that lead to the decision of shortening the vote's duration.

```

Comment:
> unless the Steering Council agrees to lift waiting requirements for situations that require prompt resolutions

Where is this coming from? Was there a case in the past where this was needed? You can instead mention that the vote can be resolved early if a majority of the entire team size is reached (e.g. 12+ votes for yes already with a maintainer team size of 23)